### ISSUE TITLE
Custom loss function for computation graph

### ISSUE THREAD

**Issue by agnibrainhack** [tensorflow/tensorflow#20643]
I want to evaluate my model in android . But it has a custom loss function similar to the one used in popular siamese neural network. How can I use TensorflowInferenceInterface in android to achieve it.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by agnibrainhack** [comment]
Os- Android
Distribution - Api 25 Nougat
Tensorflow installed from- Gradle dependency
Tensorflow version- 1.8

**Comment by tensorflowbutler** [comment]
Nagging Assignee @skye: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by agnibrainhack** [comment]
Still waiting

**Comment by skye** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new). Thanks!
